ipsec_tunnels.rst

```diff
@@ -1,14 +1,50 @@
+.. _ipsec_tunnels-section:
+
 ==============
-IPSec tunnels
+IPsec tunnels
 ==============
 
-.. warning::
+IPsec tunnels, are a crucial component of modern network security. 
+These tunnels provide a secure and encrypted communication pathway over the internet or any untrusted network, ensuring the confidentiality and integrity of data in transit. 
 
-   This feature is still under development and does not have a user interface yet. It can currently only be configured through the command line.
+IPsec (IP Security) protocol is the ‘de facto’ standard in VPN tunnels, it’s typically used to create net to net tunnels and it’s supported from all manufacturers. 
+You can use this protocol to create VPN tunnels between a NethSecurity and a device from another manufacturer as well as VPN tunnels between 2 NethSecurity.
 
+NethSecurity by default uses Route-Based VPNs, so each tunnel rely on a specific tun type device. 
+
+Configuration
+-------------
+The configuration of an IPsec tunnel includes 2 peers which we will call A and B which can be:
+
+* 1 Nethsecurity + 1 third-party firewall
```

Comment:
```suggestion
* 1 Nethsecurity and 1 third-party firewall
```

Reply:
Prefer "one" to 1, two to 2, etc if less than 10.